cvat/settings/base.py

```diff
@@ -490,3 +489,4 @@ def add_ssh_keys():
 
 USE_CACHE = True
 
+NOTIFY_JOBFLOW_URL = os.getenv('NOTIFY_JOBFLOW_URL', 'http://human-cvat:5001/notify')
```

Comment:
Please, set JOBFLOW_URL instead of particular endpoints in config. The reason is - that if you have multiple endpoints for job flow, you'll have to set all of them in the config and it could be error-prone. In the case of one variable(containing just domain + port) - you have to change just it. Nothing more

Setting here JOBFLOW_URL and particular endpoint in the place where you use it